README.md

```diff
@@ -111,6 +111,10 @@ To run tests, run the following command:
 
 - N/A
 
+## Visit my page
+
+- click on this link:  https://david30-maker.github.io/desktop-version/
```

Comment:
- [ ] [OPTIONAL] I highly suggest you put the link under the section `Live Demo` to stick to the README template. 😎️

<img src="https://user-images.githubusercontent.com/17243325/215470816-d6dc1369-5885-40cf-8352-95b5566f167a.png" width="300">